### PR TITLE
[13.0][MIG] purchase_delivery_split_date

### PR DESCRIPTION
**Pull request by ps-tubtim**
Standard Migration

### CODE REVIEW

**Review by HaraldPanten — APPROVED** [review]
Functional review. Tested in local environment 👍 @ValentinVinagre Could you review, please?

File: purchase_delivery_split_date/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (ValentinVinagre) [inline purchase_delivery_split_date/__init__.py]: change http -> https, in all documents.

File: purchase_delivery_split_date/models/purchase.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
+# Copyright 2014-2016 Numérigraphe SARL
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
```
Comment (ValentinVinagre) [inline purchase_delivery_split_date/models/purchase.py]: delete it if don't used.

**Review by ValentinVinagre — APPROVED** [review]
LGTM, only some comments.

**Comment by ps-tubtim** [comment]
@ValentinVinagre Thank you, All resolved.

**Pull request merged**